### PR TITLE
Updating node version for CI

### PR DESCRIPTION
**Pull request by muenzpraeger**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2
 # Defining default values for all jobs
 defaults: &defaults
     docker:
-        - image: circleci/node:8.9.4-browsers
+        - image: circleci/node:9.8.0-browsers
```
Comment (trevor-bliss) [inline .circleci/config.yml]: Why this version? From what I can tell LTS went from `8.15.1` to `10.13.0`. Current LTS is `10.15.3`. We should use one of those versions.

Comment (muenzpraeger) [reply .circleci/config.yml]: Yup. This was the “quick” solution. Just got word back from the CLI team what node version they support. Will also update the node version in package.json for alignment.

Comment (muenzpraeger) [reply .circleci/config.yml]: Updated to `circleci/node:lts-browsers` - that way we get automated maintenance once the LTS version moves forward. Also updated package.json to reflect new versions.

File: package.json [hunk 2]
```diff
@@ -22,15 +22,15 @@
         "url": "git+https://github.com/trailheadapps/ebikes-lwc"
     },
     "engines": {
-        "node": ">= 8.9.4",
-        "npm": ">= 5.7.1"
+        "node": ">= 10.13.0",
+        "npm": ">= 6.5.0"
```
Comment (kevinv11n) [inline package.json]: FYI These are above what we’ve required for lwc itself. 

What do sfdx and our VS Code extensions ship and require?

Comment (muenzpraeger) [reply package.json]: 8.9.4 is too low for the current CLI, the `sfdx-cli` plugin fails with that. As the CLI team recommends using LTS we're switching to that now, as 8.x is in maintenance mode.

The version change is not necessary for anything CLI related (yet), but it creates alignment.

**Comment by muenzpraeger** [comment]
Superseeded by #39

**Pull request closed without merging**